README.md

```diff
@@ -20,18 +20,25 @@ Features:
 * Leverages SpikeExtractor to support conversion from a number or proprietary formats.
 
 ## Installation
-To install **nwb-conversion-tools** directly in an existing environment:
-```
-$ pip install nwb-conversion-tools
+To install the latest stable release of **nwb-conversion-tools** though PyPI, type:
+```shell
+pip install nwb-conversion-tools
 ```
 
-Alternatively, to clone the repository and set up a conda environment, do:
+For more flexibility we recommend installing the latest version directly from GitHub. The following commands create an environment with all the required dependencies and the latest updates:
+
+```shell
+git clone https://github.com/catalystneuro/nwb-conversion-tools
+cd nwb-conversion-tools
+conda env create -f make_env.yml
+conda activate nwb_conversion_env
 ```
-$ git clone https://github.com/catalystneuro/nwb-conversion-tools
-$ cd nwb-conversion-tools
-$ conda env create -f make_env.yml
-$ conda activate nwb_conversion_env
-$ pip install .
+Note that this will install the package in [editable mode](https://pip.pypa.io/en/stable/cli/pip_install/#editable-installs). 
+
+Finally, if you prefer to avoid `conda` altogether, the following commands provide a clean installation within the current environment:
+```shell
+git clone https://github.com/catalystneuro/nwb-conversion-tools
+pip install . 
```

Comment:
```suggestion
pip install git+https://github.com/catalystneuro/nwb-conversion-tools.git@master
```

Reply:
We demonstrate above how to do a standard git clone and local installation; this is the final option to allow people to install the most up-to-date unreleased version without `conda` or having to manage the action clone.